cdn/signUrl/SignUrlSample/Program.cs

```diff
@@ -0,0 +1,128 @@
+﻿// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+using CommandLine;
+using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace SignUrlSample
+{
+    [Verb("signurls")]
+    class SignedUrlArgs
+    {
+        [Value(0, HelpText = "Url to sign", Required = true)]
+        public string Url { get; set; }
+
+        [Option('k', "key-name",
+            HelpText = "The name of the key to use when encrypting url")]
+        public string KeyName { get; set; }
+
+        [Option('p', "key-path",
+            HelpText = "The path to the key to use when encrypting url")]
+        public string KeyPath { get; set; }
+
+        [Option('e', "expiration",
+            HelpText = "Url expiration in UTC formatted as: YYYY-MM-DDThh:mm:ss")]
+        public DateTime Expiration { get; set; }
+    }
+
+    public class Program
+    {
+        static int Main(string[] args)
+        {
+            string signedUrl = null;
+            Parser.Default.ParseArguments<SignedUrlArgs>(args).MapResult(
+                uargs =>
+                    signedUrl = CreateSignedUrl(
+                                    uargs.Url,
+                                    uargs.KeyName,
+                                    File.ReadAllText(uargs.KeyPath, Encoding.UTF8),
+                                    uargs.Expiration),
+                err => string.Empty);
+
+            if (string.IsNullOrEmpty(signedUrl))
+            {
+                return 1;
+            }
+
+            Console.WriteLine($"Signed URL: {signedUrl}");
+            return 0;
+        }
+
+        // [START CreateSignedUrl]
```

Comment:
On which web page will this code appear?

Reply:
likely `/cdn/docs/signed-urls` (doesn't exist yet)